.gitignore

```diff
@@ -1 +1,3 @@
 poetry.lock
+
+__pycache__/

```

Comment:
Changes unrelated to the issue at hand are best submitted in separate PRs. In this case, however, I'd refrain because these kinds of ignore directives are better placed in a `~/.gitignore` file on your own workstation, so that they will apply to **all** projects that you interact with. In summary, please remove this change from this PR. 😊

Reply:
cool! I didn't know a person could do that.